nixos/modules/virtualisation/containers.nix

```diff
@@ -724,10 +724,9 @@ in
             ${optionalString cfg.autoStart ''
               AUTO_START=1
             ''}
-            EXTRA_NSPAWN_FLAGS="${(mkBindFlags cfg.bindMounts) +
-              (if cfg.extraFlags != null && cfg.extraFlags != []
-               then '' ${" " + concatStringsSep " " cfg.extraFlags}''
-               else "")}"
+            EXTRA_NSPAWN_FLAGS="${mkBindFlags cfg.bindMounts +
+              optionalString (cfg.extraFlags != [])
+                ''${" " + concatStringsSep " " cfg.extraFlags}''}"
```

Comment:
Look like `''${...}''` can now be replaced by `( ... )` ;)

Reply:
good point